sklearn/feature_selection/from_model.py

```diff
@@ -173,6 +173,10 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
         Otherwise train the model using ``fit`` and then ``transform`` to do
         feature selection.
 
+    max_n_features : int, between 0 and number of features, default None.
+        If max_n_features is provided, first n most important features are
```

Comment:
max_n_features should be wrapped in double-backticks; or you can just say "if provided"